src/macaron/slsa_analyzer/git_url.py

```diff
@@ -907,3 +907,79 @@ def is_empty_repo(git_obj: Git) -> bool:
         return False
     except GitCommandError:
         return True
+
+
+def is_commit_hash(version_str: str) -> bool:
```

Comment:
Could we rename this variable to make it more generic? From the doc string, I don't think it must be a version string.